cmd2/cmd2.py

```diff
@@ -176,10 +158,37 @@ def cat_decorator(func):
     return cat_decorator
 
 
+def _get_command_arg_list(to_parse: Union[Statement, str], preserve_quotes: bool) -> List[str]:
```

Comment:
I recommend moving this helper function to be a standalone function in parsing.py